x-pack/plugins/secops/public/components/timeline/body/column_headers/header/index.tsx

```diff
@@ -33,11 +33,16 @@ const HeaderContainer = styled(EuiFlexGroup)<{ width: string }>`
   width: ${({ width }) => width};
 `;
 
+const HeaderFlexItem = styled(EuiFlexItem)<{ width: string }>`
+  width: ${({ width }) => width};
```

Comment:
do we need to specify if it is px or % or em or rm?

Reply:
the api is not opinionated about that, so either will work here